src/structs/field-shorthand.md

```diff
@@ -34,4 +33,11 @@ impl Person {
 }
 ```
     
+
+<details>
+
+* Point out the syntax that is used inside the `new` function. To show a comparison you can also rewrite the struct definition using non-shorthand syntax.
```

Comment:
```suggestion
```

Reply:
It seems to me that this bullet point doesn't say anything new — at this point, students have already seen a method or two.